app/getBooks.py

```diff
@@ -0,0 +1,36 @@
+from db.book import Book
+
+from flask import (
+    Blueprint, flash, redirect, render_template, request, url_for
+)
+
+bp = Blueprint('getbook', __name__, url_prefix='/getbook')
+
+
+
+@app.route("/", methods=['GET', 'POST'])
+def getBook():
+    #request.method and request.form
+        #request.method --> going to submit the POST method
+        #then it will get request.form
+    bookListJson == ""
+    if request.method == 'GET':
+        if request.form['BookName']:
```

Comment:
i think if the form does not have 'BooName' value then this would prob give us an error since the dict would not have a key of 'BookName'
u can take a look at this to find out more.
https://scotch.io/bar-talk/processing-incoming-request-data-in-flask